### PR TITLE
Body divider slot5.0

### PR DESCRIPTION
**Pull request by bennettclark**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [x] Include a change request file using `$ npm run change`

#### Description of changes

Cherry picked from commits to https://github.com/OfficeDev/office-ui-fabric-react/pull/6194

### CODE REVIEW

File: common/changes/@uifabric/variants/bodyDividerSlot5.0_2018-09-04-18-34.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/variants",
+      "comment": "Change bodyDivider value from neutralTertiaryAlt back to neutralLight",
+      "type": "minor"
```
Comment (cliffkoh) [inline common/changes/@uifabric/variants/bodyDividerSlot5.0_2018-09-04-18-34.json]: I don't see a change for variants in this PR...

File: common/changes/@uifabric/styling/bodyDividerSlot5.0_2018-09-04-18-34.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@uifabric/styling",
+      "comment": "Change bodyDivider value from neutralTertiaryAlt back to neutralLight",
+      "type": "minor"
```
Comment (cliffkoh) [inline common/changes/@uifabric/styling/bodyDividerSlot5.0_2018-09-04-18-34.json]: looks like a patch change to me

File: common/changes/office-ui-fabric-react/bodyDividerSlot5.0_2018-09-04-18-34.json [hunk 3]
```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "office-ui-fabric-react",
+      "comment": "Change bodyDivider value from neutralTertiaryAlt back to neutralLight",
+      "type": "minor"
```
Comment (cliffkoh) [inline common/changes/office-ui-fabric-react/bodyDividerSlot5.0_2018-09-04-18-34.json]: looks like a patch change to me

File: common/changes/@uifabric/styling/bodyDividerSlot5.0_2018-09-04-18-34.json (hunk 2 above)
Comment (bennettclark) [reply common/changes/@uifabric/styling/bodyDividerSlot5.0_2018-09-04-18-34.json]: updated!

File: common/changes/@uifabric/variants/bodyDividerSlot5.0_2018-09-04-18-34.json (hunk 1 above)
Comment (bennettclark) [reply common/changes/@uifabric/variants/bodyDividerSlot5.0_2018-09-04-18-34.json]: Removed log!

File: common/changes/office-ui-fabric-react/bodyDividerSlot5.0_2018-09-04-18-34.json (hunk 3 above)
Comment (bennettclark) [reply common/changes/office-ui-fabric-react/bodyDividerSlot5.0_2018-09-04-18-34.json]: Updated

**Comment by phkuo** [comment]
@bennettclark yeah those quotes are part of the literal value of the variable itself, they get printed into the ouptut CSS, so they're exempt from whatever style guidelines we have

**Pull request merged**